File: trip-kit-booking/src/main/java/com/skedgo/tripkit/booking/mybookings/MyBookingsApi.java
package com.skedgo.tripkit.booking.mybookings;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface MyBookingsApi {
    @GET("booking")
    Observable<MyBookingsResponse> fetchMyBookingsAsync(
        @Query("first") int first,
        @Query("max") int pageSize,
        @Query("bsb") int bsb
    );
}

File: trip-kit-booking/src/main/java/com/skedgo/tripkit/booking/mybookings/MyBookingsApi.kt
package com.skedgo.tripkit.booking.mybookings

import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Query

interface MyBookingsApi {
    @GET("booking")
    fun fetchMyBookingsAsync(
        @Query("first") first: Int,
        @Query("max") pageSize: Int,
        @Query("bsb") bsb: Int
    ): Observable<MyBookingsResponse?>?
}
